src/Depressurizer/MainForm.cs

```diff
@@ -3626,6 +3626,28 @@ private void namedescendingToolStripMenuItem_Click(object sender, EventArgs e)
             lstCategories.Sort();
         }
 
+        /// <summary>
+        /// This menu item will open a dialog box with a random game from the currently filtered set of games.  If the game
+        /// is a Steam game it will show the banner, otherwise just the name of the game.  This functionality is to help
+        /// users with large game libraries decide which game to play, should they feel overwhelmed by their options.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void RandomGameSelectionToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            if (!ProfileLoaded) return;
```

Comment:
Please adhere to the code style, similar constructs are above this part of the code.

Reply:
```
if (!ProfileLoaded) {
    return;
}
```